app/assets/javascripts/backbone/views/loan_questionnaires_view.coffee

```diff
@@ -86,10 +87,17 @@ class MS.Views.LoanQuestionnairesView extends Backbone.View
   editDocument: (e) ->
     # Fire a global event. Consider refactoring this in the style of notifyExpandListeners above.
     Backbone.trigger 'LoanQuestionnairesView:edit', @
+    @flexibleTextArea()
 
   # Sets up the dirtyForm plugin on the questionnaire form.
   # We need to do this on a focus event because the form is not always visible when the page loads,
   # and dirtyForms doesn't seem to work with a hidden form. The focus event can only be fired when the
   # form is visible. Also, calling .dirtyForms() on each focus event doesn't seem to cause any issues.
   setupDirtyForm: (e) ->
     @$('.questionnaire-form form').dirtyForms()
+
+  flexibleTextArea: (e) ->
+    $('.answer-wrapper textarea')
+    .on 'change keyup keypress keydown paste cut clear', ->
+      $(this).height(0).height @scrollHeight
+    .change()
```

Comment:
Traditionally, we place the events inside the events function earlier in the document. It's a Backbone convention.

````
  events:
    'ajax:error': 'submitError'
    'confirm:complete .linked-document [data-action="delete"]': 'removeLinkedDocument'
    'click .edit-action': 'editDocument'
    'focus .questionnaire-form form': 'setupDirtyForm'
    'tree.open': 'notifyExpandListeners'
    'change .answer-wrapper textarea': 'flexibleTextArea'
    'keyup .answer-wrapper textarea': 'flexibleTextArea'
    'keydown .answer-wrapper textarea': 'flexibleTextArea'
    'paste .answer-wrapper textarea': 'flexibleTextArea'
    'cut .answer-wrapper textarea': 'flexibleTextArea'
    'clear .answer-wrapper textarea': 'flexibleTextArea'

flexibleTextArea: ->
   $('.answer-wrapper textarea').height(0).height(@scrollHeight)
````
However, this breaks things (specifically the flexibleTextArea function breaks things). So I'm not sure what can be done.